src/config/useToast.tsx

```diff
@@ -0,0 +1,189 @@
+import * as React from "react"
+import { ToastActionElement } from "../components/Molucules/Toast/ToastAction"
+import { ToastProps } from "../components/Molucules/Toast/Toast"
+ 
+ 
```

Comment:
@siroshan  Let's move this custom hook inside the relevant component file & export it from the index file so that everything's in one place—easy to copy & move components that way!